Source/WebCore/platform/graphics/ImageBuffer.h

```diff
@@ -155,6 +155,10 @@ class ImageBuffer : public ThreadSafeRefCounted<ImageBuffer, WTF::DestructionThr
     AffineTransform baseTransform() const { return m_backendInfo.baseTransform; }
     size_t memoryCost() const { return m_backendInfo.memoryCost; }
     size_t externalMemoryCost() const { return m_backendInfo.externalMemoryCost; }
+    void setBackendInfo(ImageBufferBackend::Info& parameters)
```

Comment:
Should either be a `const ImageBufferBackend::Info&` or `ImageBufferBackend::Info&&`. Here, I suggest `ImageBufferBackend::Info&&`

Reply:
I agree. Changed to `ImageBufferBackend::Info&&`.